typescript/simple-mercari-web/src/App.css

```diff
@@ -26,26 +51,30 @@ body{
   justify-content: left;
   font-size: calc(10px + 1vmin);
   color: white;
-  padding: 4vh;
 }
 
-.Listing input[type="text"]{
+.Form input[type="text"]{
   margin-right: 10px;
+  margin-bottom: 20px;
   padding: 10px;
   background-color: #303337;
   border: 1px solid #aaaaaa;
   border-radius: 5px;
   color: #ffffff
 }
+.Form label{
+  margin-bottom: 10px;
+}
 
-.Listing button {
+button {
```

Comment:
this is just trivial, shall we add a hover effect on the button cuz it will look cool...! ( ̀⌄ ́ )

Reply:
Done!